### ISSUE TITLE
[0.65.0, Android]  Could not find com.henninghall.android:NumberPickerView:1.1.5.

### ISSUE THREAD

**Issue by efstathiosntonas** [henninghall/react-native-date-picker#362]
**Describe the bug**
After upgrading to react-native 0.65.0 (followed react-native-upgrade-helper, updated from 0.64.2) it throws:

```
 Could not find com.henninghall.android:NumberPickerView:1.1.5.
```

detailed:
```
 > Could not find com.henninghall.android:NumberPickerView:1.1.5.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/henninghall/android/NumberPickerView/1.1.5/NumberPickerView-1.1.5.pom
       - file:/Users/stathis/.m2/repository/com/henninghall/android/NumberPickerView/1.1.5/NumberPickerView-1.1.5.pom
       - file:/Users/stathis/WebstormProjects/xxxxxxxx/app/node_modules/react-native/android/com/henninghall/android/NumberPickerView/1.1.5/NumberPickerView-1.1.5.pom
       - file:/Users/stathis/WebstormProjects/xxxxxxxx/app/node_modules/jsc-android/dist/com/henninghall/android/NumberPickerView/1.1.5/NumberPickerView-1.1.5.pom
       - https://dl.google.com/dl/android/maven2/com/henninghall/android/NumberPickerView/1.1.5/NumberPickerView-1.1.5.pom
       - file:/Users/stathis/WebstormProjects/xxxxxxx/app/node_modules/@bugsnag/react-native/android/com/henninghall/android/NumberPickerView/1.1.5/NumberPickerView-1.1.5.pom
     Required by:
         project :app > project :react-native-date-picker

```

**Expected behavior**
build on Android 

**Smartphone (please complete the following information):**
 - OS: Android
 - React Native version [0.65.0]
 - react-native-date-picker version [3.3.2]

build.gradle:
```
buildscript {
    ext {
        buildToolsVersion = "30.0.2"
        minSdkVersion = 21
        compileSdkVersion = 30
        targetSdkVersion = 30
        supportLibVersion = "29.0.0"
        kotlinVersion = "1.5.0"
        ndkVersion = "20.1.5948944"
    }
    repositories {
       google()
       mavenCentral()
       }
    dependencies {
        classpath 'com.android.tools.build:gradle:4.2.1'
        classpath 'com.google.gms:google-services:4.3.8'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
        classpath "com.bugsnag:bugsnag-android-gradle-plugin:5.+"
    }
}

allprojects {
     repositories {
     mavenCentral()
     mavenLocal()


        maven {
            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
            url("$rootDir/../node_modules/react-native/android")
        }
        maven {
            // Android JSC is installed from npm
            url("$rootDir/../node_modules/jsc-android/dist")
        }
      google()
    }
    subprojects {
        project.configurations.all {
            resolutionStrategy.eachDependency { details ->
                if (details.requested.group == 'com.android.support'
                        && !details.requested.name.contains('multidex') ) {
                    details.useVersion "29.0.0"
                }
            }
        }
        afterEvaluate {
            project -> if (project.hasProperty("android") || project.plugins.hasPlugin('android-library')) {
                android {
                    compileSdkVersion rootProject.ext.compileSdkVersion
                    buildToolsVersion rootProject.ext.buildToolsVersion
                }
            }
        }
    }
}

```

**Comment by LuisRego** [comment]
Having the same issue 😢

**Comment by husseinAbdulnabi** [comment]
same issue

**Comment by henninghall** [comment]
Can you try to add `jcenter()` to repositories part in `android/build.gradle`, did it solve it?

```
repositories {
        jcenter()
        ...
}
```

**Comment by husseinAbdulnabi** [comment]
@henninghall yea this works, thank you

**Comment by henninghall** [comment]
This fix is no longer needed in version 3.4.0

**Comment by devanshsadhotra25** [comment]
@henninghall  issue still exists in v4.0.0
